cmake/kokkos_functions.cmake

```diff
@@ -954,6 +954,58 @@ int main()
   set(${_VAR} ${_RET} CACHE STRING "CXX compiler supports building CUDA")
 endfunction()
 
+# this function is provided to easily select which files use nvcc_wrapper:
+#
+#       COMPILER    --> do check for compiler
+#       LINKER      --> do check for linker
+#       FLAGS       --> flags to check
+#
+function(kokkos_check_flags)
+  cmake_parse_arguments(INP "COMPILER;LINKER" "" "FLAGS;LANGUAGE" ${ARGN})
```

Comment:
Do you mean for `LANGUAGE` to allow multiple value?

Reply:
nope. good catch
```suggestion
  cmake_parse_arguments(INP "COMPILER;LINKER" "LANGUAGE" "FLAGS" ${ARGN})
```